scripts/common/get_logs.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+__author__="Amandeep Singh Arora"
+__version__="1.0"
+
+> /tmp/fledge_syslog.log
+
+exec 2<&-
+exec 2<>/tmp/fledge_syslog.log
```

Comment:
We should avoid the file naming with Product prefix.

Reply:
Changed "fledge_" to "fl_" throughout the file.